flag_int64_slice.go

```diff
@@ -135,7 +141,48 @@ func lookupInt64Slice(name string, set *flag.FlagSet) []int64 {
 		if err != nil {
 			return nil
 		}
+		// extract default value from the flag
+		var defaultVal []int64
+		for _, v := range strings.Split(f.DefValue, ",") {
+			iV, _ := strconv.ParseInt(v, 10, 64)
```

Comment:
Request: Even if we're confident that this parse should always work, I'd rather panic explicitly than silently ignore the error. Panics are for developer errors.

The reason for this is that if we or the user somehow make a mistake that allows a non-int to end up here, the current code will silently set this value to `0` and carry on into the territory of behavior we haven't planned for. If we panic instead, we get a clear message back to the end user that something is actually catastrophically wrong, and we stop the program.

```go
iV, err := strconv.ParseInt(v, 10, 64)
if err != nil {
	panic(err)
}
```

Reply:
I agree. I just noticed the other PR fixing this in the master and I would love to get it done right here.